plugins/disk_size/disk_sizes.rb

```diff
@@ -0,0 +1,79 @@
+require 'json'
+require 'redis'
+require 'time'
+# variable to store the default time difference from start and endTime
+# = secs*mins*hours*day
+DAYS = 1
+TIME_INTERVAL = 60 * 60 * 24 * DAYS
+
+# disk size plugin
+class DiskSize
+  def self.register
+    puts 'DiskSize registering'
+    # 1. create tables in DB
+    # 2. register itself to the plugin registry
+  end
+
+  def self.collect
+    puts 'DiskSize collecting'
+    # collect data into DB
+  end
+
+  def report(node, start_time, end_time)
+    # defaults if empty strings are sent in
+    end_time = Time.now if end_time == ''
+    start_time = Time.now - TIME_INTERVAL if start_time == ''
+    range = start_time.to_i..end_time.to_i
+
+    # puts 'node: ' + node
+    # puts 'start: ' + start_time.to_s
+    # puts 'end: ' + end_time.to_s
+
+    # gather data from Db here
+    # dataset = DB[:disk_size_measurement]
+    redis = Redis.new(host: ENV['REDIS_HOST'])
+    data_hash = {}
+    servers = redis.keys
```

Comment:
I don't think we want to get all keys (even when we are using a database), unless specifically requested by the user.